http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=563971

rpcbind only enables the -w option if the --enable-warmstarts
option is passed to configure.

The fact that it does not print a useful error message in this
case is an error.

Signed-off-by: sacrificial-spam-address@horizon.com
Tested-by: Anibal Monsalve Salazar <anibal@debian.org>

--- a/src/rpcbind.c	2010-01-06 16:10:17.000000000 -0500
+++ b/src/rpcbind.c	2010-01-06 16:09:57.864844553 -0500
@@ -758,13 +758,18 @@
 		case 's':
 			runasdaemon = 1;
 			break;
-#ifdef WARMSTART
 		case 'w':
+#ifdef WARMSTART
 			warmstart = 1;
 			break;
-#endif
 		default:	/* error */
 			fprintf(stderr,	"usage: rpcbind [-adhilsw]\n");
+#else
+			fprintf(stderr,	"-w: rpcbind compiled without WARMSTART support.\n");
+			/* FALLTHROUGH */
+		default:	/* error */
+			fprintf(stderr,	"usage: rpcbind [-adhils]\n");
+#endif
 			exit (1);
 		}
 	}
